plugin/helpers.go

```diff
@@ -8,8 +8,16 @@ import "github.com/mattermost/mattermost-server/model"
 type Helpers interface {
 	// EnsureBot either returns an existing bot user matching the given bot, or creates a bot user from the given bot.
 	// Returns the id of the resulting bot.
+	//
+	// Minimum server version: 5.10
 	EnsureBot(bot *model.Bot) (string, error)
```

Comment:
@ali-farooq0 and I talked about this offline and we should like to consolidate `EnsureBot` and `EnsureBotWithOptions` to just `EnsureBot`. `EnsureBot` would have the signature `(bot *model.Bot, options ...Option) (string, error)`. This is not a breaking change, because the options are optional :wink:.

Would you be open on making this change? Sorry for changing our mind on this.